### PR TITLE
qt: use brewed harfbuzz

### PR DESCRIPTION
**Pull request by paperchalice**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
- use brewed `harfbuzz`
- `ffmpeg` issue was resolved in commit https://github.com/qt/qtwebengine-chromium/commit/9a1051732da271f8abab4242b683e922399a8d5a
- use console app in test (may fix linux test)

### CODE REVIEW

File: Formula/qt.rb [hunk 1]
```diff
@@ -203,6 +204,7 @@ def install
     if OS.mac?
       cmake_args << "-DCMAKE_OSX_DEPLOYMENT_TARGET=#{MacOS.version}.0"
       config_args << "-sysroot" << MacOS.sdk_path.to_s
+      config_args << "-system-harfbuzz"
```
Comment (SMillerDev) [inline Formula/qt.rb:207]: Why not on linux?

Comment (SMillerDev) [reply Formula/qt.rb:207]: Does it break if we still pass this flag?

Comment (SMillerDev) [reply Formula/qt.rb:207]: Can we unconditionally pass this then? It makes it easier to see the state of the dependency.

**Comment by fxcoudert** [comment]
Following the recent test fix, this should allow a Linux bottle to be built and, hopefully, merged ASAP.

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**